Wazuh DB state unit tests failed

|Wazuh version|Component|Install type|
|---|---|---|
| 4.6.0-dev | Unit tests| Manager |

```
[==========] Running 1 test(s).
[ RUN      ] test_wazuhdb_create_state_json
[  ERROR   ] --- 0x658c != 0x658a
[   LINE   ] --- /root/wazuh/src/unit_tests/wazuh_db/test_wazuh_db_state.c:513: error: Failure!
[  FAILED  ] test_wazuhdb_create_state_json
[==========] Running 1 test(s).
[ RUN      ] test_wazuhdb_create_state_json
[  ERROR   ] --- 0x47f9 != 0x47f8
[   LINE   ] --- /root/wazuh/src/unit_tests/wazuh_db/test_wazuh_db_state.c:518: error: Failure!
[  FAILED  ] test_wazuhdb_create_state_json
[  ERROR   ] --- 0xdf != 0xde
[   LINE   ] --- /root/wazuh/src/unit_tests/wazuh_db/test_wazuh_db_state.c:548: error: Failure!
[  FAILED  ] test_wazuhdb_create_state_json
```

This issue seems to be caused by PR #19073, whose checks did not include the CMocka unit tests due to an unknown reason.

--- a/src/unit_tests/wazuh_db/test_wazuh_db_state.c
+++ b/src/unit_tests/wazuh_db/test_wazuh_db_state.c
@@ -510,12 +510,12 @@ void test_wazuhdb_create_state_json(void ** state) {
     cJSON* time = cJSON_GetObjectItem(metrics, "time");
 
     assert_non_null(cJSON_GetObjectItem(time, "execution"));
-    assert_int_equal(cJSON_GetObjectItem(time, "execution")->valueint, 25550);
+    assert_int_equal(cJSON_GetObjectItem(time, "execution")->valueint, 25996);
 
     cJSON* execution_breakdown = cJSON_GetObjectItem(time, "execution_breakdown");
 
     assert_non_null(cJSON_GetObjectItem(execution_breakdown, "agent"));
-    assert_int_equal(cJSON_GetObjectItem(execution_breakdown, "agent")->valueint, 17979);
+    assert_int_equal(cJSON_GetObjectItem(execution_breakdown, "agent")->valueint, 18425);
 
     cJSON* agent_time_breakdown = cJSON_GetObjectItem(execution_breakdown, "agent_breakdown");
 
@@ -545,7 +545,7 @@ void test_wazuhdb_create_state_json(void ** state) {
     assert_non_null(cJSON_GetObjectItem(agent_syscheck_time, "fim_registry"));
     assert_int_equal(cJSON_GetObjectItem(agent_syscheck_time, "fim_registry")->valueint, 221);
     assert_non_null(cJSON_GetObjectItem(agent_syscheck_time, "fim_registry_key"));
-    assert_int_equal(cJSON_GetObjectItem(agent_syscheck_time, "fim_registry_value")->valueint, 222);
+    assert_int_equal(cJSON_GetObjectItem(agent_syscheck_time, "fim_registry_value")->valueint, 223);
     assert_non_null(cJSON_GetObjectItem(agent_syscheck_time, "fim_registry_key"));
     assert_int_equal(cJSON_GetObjectItem(agent_syscheck_time, "fim_registry_value")->valueint, 223);
 

